modules/common/src/main/scala/io/circe/jackson/enrich/WithJacksonMapper.scala

```diff
@@ -0,0 +1,18 @@
+package io.circe.jackson.enrich
+
+import java.io.{File, Writer}
+
+import com.fasterxml.jackson.core.{JsonFactory, JsonParser}
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.snowplowanalytics.snowplow.enrich.common.utils.CirceUtils
+
+class WithJacksonMapper {
+  final val mapper: ObjectMapper = CirceUtils.mapper
```

Comment:
We need to note somewhere that this whole PR, all new modules are just to make this `mapper` default. Right now it looks like a huge copy-pasta if we don't point to this mapper.